mods/cnc/scripts/campaign.lua

```diff
@@ -9,16 +9,24 @@
 
 Difficulty = Map.LobbyOption("difficulty")
 
+AddPrimaryObjective = function(player, description)
+	return player.AddObjective(UserInterface.Translate(description), UserInterface.Translate("primary"), true)
+end
+
+AddSecondaryObjective = function(player, description)
+	return player.AddObjective(UserInterface.Translate(description), UserInterface.Translate("secondary"), false)
```

Comment:
translating "primary" and "secondary" seems wrong. We later use this string in `GetObjectiveDescription` which will require translators to change `new-primary-objective` and `new-secondary-objective` values along side with variables names. We don't require this from translators from anywhere else